utils/ssh/scp.go

```diff
@@ -60,7 +60,7 @@ type easyProgressUtil struct {
 }
 
 //must call DisplayInit first
-func RegisterEpu(ip string, total int) {
+func registerEpu(ip string, total int) {
 	if progressChanOut == nil {
```

Comment:
You still need to consider that the DisplayInit function is not called, so you can add init under the ssh.interface package to implement the DisplayInit function